### DISCUSSION TITLE
BlackPill only 8 analogue

### DISCUSSION THREAD

**Discussion by jeremydte** [shanteacontrols/OpenDeck#294] [Q&A]
I have built the opendeck board 3.0.1 with a Blackpill401cc and I have only 8 analogue input.
<img width="1112" alt="Capture d’écran 2024-05-20 à 13 14 32" src="https://github.com/shanteacontrols/OpenDeck/assets/72756948/e16df24f-ba40-44f0-a132-1b956c4460b4">

How can I have the full feature of the opendeck L board?

Sorry for my English.

**Comment by paradajz** [comment]
You need to load the correct firmware on it, the one called `opendeck3b.bin`. `blackpill401cc` is the default configuration.

**Reply by jeremydte** [reply]
Thank for your answer, but I've tri to load the opendeck.bin via the terminal but the board is no longer reconized (No OpenDeck board found. Please connect the board in order to use the interface.)
And If i try to opload with the configurator nothing happend, it's still the BlackpillF401cc firmware.
